Sources/Apollo/StringEncoding.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  File.swift
+//  
+//
+//  Created by abdi maye on 16/12/2020.
+//
+
+import Foundation
+
+#if os(Linux)
+enum APITextEncoding : String
+{
+    case ascii = "us-ascii"
+    case iso2022JP = "iso-2022-jp"
+    case isoLatin1 = "iso-8859-1"
+    case isoLatin2 = "iso-8859-2"
+    case japaneseEUC = "euc-jp"
+    case macOSRoman = "macintosh"
+    case utf8 = "utf8"
+    case utf16 = "utf16"
+    case utf32 = "utf32"
+
+    var encoding: String.Encoding? {
+        switch self
+        {
+        case .ascii:
+            return .ascii
+        case .iso2022JP:
+            return .iso2022JP
+        case .isoLatin1:
+           return .isoLatin1
+        case .isoLatin2:
+            return .isoLatin2
+        case .japaneseEUC:
+            return .japaneseEUC
+        case .macOSRoman:
+            return .macOSRoman
+        case .utf8:
+            return .utf8
+        case .utf16:
+            return .utf16
+        case .utf32:
+            return .utf32
+        }
```

Comment:
Don't really know how else to do this. This is also no exhaustive as it's quite tricky to figure out the encoding properties in swift https://developer.apple.com/documentation/swift/string/encoding and the actual names https://www.iana.org/assignments/character-sets/character-sets.xhtml 😫 

Any help/pointers would be much appreciated!

Reply:
Actually, I just looked to see where we're using this and...we aren't. I think I can delete this method, which will remove the need for all of this.

Reply:
Just opened #1579, once that gets merged I think you should be able to look at the merge conflicts.